### ISSUE TITLE
Disk usage and memory inconsistency in minikube start

### ISSUE THREAD

**Issue by aps831** [kubernetes/minikube#9115]
minikube version: v1.12.3

The flags for setting disk size and memory have the same usage other than for the default value:
 
```console
minikube start --help [snipped]
     --disk-size='20000mb': Disk size allocated to the minikube VM (format: <number>[<unit>], where unit = b, k, m or g).   
     --memory='': Amount of RAM to allocate to Kubernetes (format: <number>[<unit>], where unit = b, k, m or g).
```

Specifying no unit works as expected:
```console
minikube start --driver=virtualbox --memory=8192 --disk-size=8192
[snipped]
Creating virtualbox VM (CPUs=2, Memory=8192MB, Disk=8192MB) ..
```

Specifying a unit of ```m``` works for disk-size only:
```console
minikube start --driver=virtualbox --memory=8192m --disk-size=8192m
[snipped]
Creating virtualbox VM (CPUs=2, Memory=3900MB, Disk=8192MB) ...
```

Specifying a unit of ```mb``` works for disk-size only:
```console
minikube start --driver=virtualbox --memory=8192mb --disk-size=8192mb
[snipped]
Creating virtualbox VM (CPUs=2, Memory=3900MB, Disk=8192MB) ...
```

**Comment by priyawadhwa** [comment]
Hey @aps831 thanks for pointing out this inconsistency! If you, or anyone else, would be interested in fixing this, feel free to assign yourself to this issue by commenting /assign.

**Comment by Dean-Coakley** [comment]
@aps831 I'm struggling to reproduce this issue, can you try again on minikube 1.13?

**Comment by aps831** [comment]
Will do.  It will be a couple of days before I get a chance though.

**Comment by aps831** [comment]
Working on minikube 1.13.  Fixed by #9033
